docs/Kubernetes/deployingk8s-aks.md

```diff
@@ -0,0 +1,117 @@
+## Prerequisites
+
+- Docker Desktop w/ kubectl
+- Azure CLI (v2.24.0+)
+- Helm CLI (v3.5+)
+
+# AKS Deployment
+
+## Create SSH Key
+```
+ssh-keygen -t rsa -b 2048
+```
+Note the location as you will need the public key (`.pub` file) in the next step. 
+
+## Deploy AKS
+
+Login using `az login` if you haven't already and make sure the correct subscription is set as default using `az account set`
+``` bash
+az group create --name <your-resource-group-name> --location eastus
+```
+
+```
+az deployment group create --resource-group <your-resource-group-name> --template-file aks.json
+```
+You will be prompted for a name for the AKS Cluster, the linux user admin name (i.e. your name), and the public key (`.pub` file) you generated in step 1. This takes about ~5 min. Note the `fqdnSuffix` in the "outputs" section of the JSON response (i.e. `westus2.cloudapp.azure.com`) when complete. This will be needed for the updating the commonName in the `values.yaml` file.
```

Comment:
Hmm. "Step 1". By that do you mean the Create SSH Key?

Reply:
Yes, got rid of steps , can change.